src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -879,7 +889,23 @@ void NonClientIslandWindow::_UpdateFrameMargins() const noexcept
         //  at the top) in the WM_PAINT handler. This eliminates the transparency
         //  bug and it's what a lot of Win32 apps that customize the title bar do
         //  so it should work fine.
-        margins.cyTopHeight = -frame.top;
+        //
+        // Note #3 (circa late 2022): It seems like we can just set this to 0,
+        //  and have it work. This is LOAD-BEARING. By having the titlebar a
+        //  totally empty rect, DWM will know that we don't have the traditional
+        //  titlebar, and will use NCHITTEST to determine where to place the
+        //  Snap Flyout. The drag rect will handle that.
+        //
+        //  FURTHERMORE: If we leave the titlebar visible AT ALL, then a
+        //  transparent titlebar (theme.tabRow.background:#ff00ff00 for example)
+        //  will allow the DWM titlebar to be visible, which will look insane.
+        //  EVEN MORE SO: Mica + "show accent color on title bars" will _always_
+        //  show the accent-colored strip of the titlebar, even on top of the
+        //  Mica.
+        //
+        //  So REALLY REALLY get rid of the titlebar entirely.
+
+        margins.cyTopHeight = 0;

```

Comment:
I **need** you to test this on Windows 10 before I am comfortable signing off on it. There was a very good reason we kept this -- look for an e-mail titled "Terminal window & transparent background". It's about `DwmExtend`, but... these things are very, very related!

Reply:
(as discussed, I did)